docs/en/seatunnel-engine/web-ui.md

```diff
@@ -0,0 +1,33 @@
+# Apache SeaTunnel Web UI Documentation
+
+## Overview
+
+The Web UI of Apache SeaTunnel offers a user-friendly interface for monitoring and managing SeaTunnel jobs. Through the Web UI, users can view real-time information on currently running jobs, finished jobs, and the status of worker and master nodes within the cluster. The main functional modules include Jobs, Workers, and Master, each providing detailed status information and operational options to help users efficiently manage and optimize their data processing workflows.
+![workflow.png](../../images/ui/overview.png)
+
+## Jobs
```

Comment:
Quick question: How can I access it?

Reply:
Sorry, images was ignored, I've just resolved it